gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/GaaSJobObservabilityProducerTest.java

```diff
@@ -136,7 +137,7 @@ public void testCreateGaaSObservabilityEventWithFullMetadata() throws Exception
     Assert.assertEquals(event.getDatasetsMetrics().get(1).getEntitiesWritten(), Long.valueOf(dataset2.getRecordsWritten()));
     Assert.assertEquals(event.getDatasetsMetrics().get(1).getBytesWritten(), Long.valueOf(dataset2.getBytesWritten()));
     Assert.assertEquals(event.getDatasetsMetrics().get(1).getSuccessfullyCommitted(), Boolean.valueOf(dataset2.isSuccessfullyCommitted()));
-    Assert.assertEquals(event.getJobProperties(), "{\"gobblin.flow.sourceIdentifier\":\"sourceNode\",\"gobblin.flow.destinationIdentifier\":\"destinationNode\",\"user.to.proxy\":\"newUser\",\"flow.executionId\":\"1681242538558\"}");
+    JsonParser.parseString(event.getJobProperties()); // Should not throw
```

Comment:
what are we validating not to throw - the `getJobProperties()` call or the JSON parsing?

Reply:
JSON parsing that the string is well formed from a properties object to deser